scout/adapter/mongo/filter.py

```diff
@@ -4,17 +4,9 @@
 from bson.objectid import ObjectId
 from flask import url_for
 
-LOG = logging.getLogger(__name__)
+from scout.constants.variant_tags import VARIANTS_TARGET_FROM_CATEGORY
```

Comment:
We normally import to the main constants, and export them from there, dont we?

Reply:
Ah OK, I'll change it like that!